src/modal/modal.service.ts

```diff
@@ -99,6 +99,7 @@ export class ModalService {
 				modalLabel: data.label || data.modalLabel,
 				modalTitle: data.title || data.modalTitle,
 				modalContent: data.content || data.modalContent,
+				hasScrollingContent: data.hasScrollingContent,
```

Comment:
If we need explicit `null`s we should be sure to use `|| null` here as a fallback